### PR TITLE
Add katello-packaging to pull request scanner

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "hammer_cli_foreman_discovery",
     "katello",
     "katello_installer",
+    "katello-packaging",
```
Comment (domcleal) [inline puppet/modules/slave/manifests/init.pp]: Use an underscore, it has to match the filename below.

File: puppet/modules/slave/templates/test_pull_requests_katello_packaging.json.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+    "proxy_host": null,
+    "proxy_port": null,
+    "jenkins_host": "http://ci.theforeman.org/",
+    "jenkins_url": "http://ci.theforeman.org/",
+    "hub_config": "~/.config/hub",
+    "github_user": "katello",
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_katello_packaging.json.erb]: This *may* need to be uppercase to match the org name, but I'm not certain.

**Comment by ehelms** [comment]
Updated - changed to uppercase and the dash to underscore

**Comment by domcleal** [comment]
Thanks @ehelms.

**Pull request merged**